metricflow-semantics/metricflow_semantics/model/semantics/linkable_spec_resolver.py

```diff
@@ -218,13 +219,17 @@ def __init__(
         # If no metrics are specified, the query interface supports querying distinct values for dimensions, entities,
         # and group by metrics.
         linkable_element_sets_for_no_metrics_queries: List[LinkableElementSet] = []
+        # Temp: separate loop to track latency for linkable metrics. TODO: combine with loop below.
         for semantic_model in semantic_manifest.semantic_models:
-            linkable_element_sets_for_no_metrics_queries.append(self._get_elements_in_semantic_model(semantic_model))
             linkable_element_sets_for_no_metrics_queries.append(
                 self.get_joinable_metrics_for_semantic_model(
                     semantic_model, SemanticModelJoinPath(left_semantic_model_reference=semantic_model.reference)
                 )
             )
+        logger.info(f"Building valid linkable metrics took: {time.time() - linkable_metrics_start_time:.2f}s")
```

Comment:
:( we need to figure out how to add stuff to `extra` in MF logs.... having it as a field in DD would allow us to easily calculate all sorts of stats on it